### PR TITLE
Navigating and Displaying Participants List

### PR DESCRIPTION
**Pull request by saicheongkam**

### CODE REVIEW

File: Frontend/app/shared/participant.ts [hunk 1]
```diff
@@ -61,4 +61,20 @@ export class Participant{
         }
         return maxWest;
     }
+
+    getNumStories() : string {
+        var numStories = "";
```
Comment (saicheongkam) [inline Frontend/app/shared/participant.ts]: return "(" + this.places.length + ")";

File: Frontend/app/shared/participant.ts [hunk 2]
```diff
@@ -61,4 +61,20 @@ export class Participant{
         }
         return maxWest;
     }
+
+    getNumStories() : string {
```
Comment (saicheongkam) [inline Frontend/app/shared/participant.ts:65]: return "(" + this.places.length + " stories)"; should be a simpler way to do this. :)

Comment (saicheongkam) [reply Frontend/app/shared/participant.ts:65]: Also add conditions so that it displays 'story' instead of 'stories' if there is only 1.

Comment (danielelkington) [reply Frontend/app/shared/participant.ts:65]: Done

**Pull request merged**